src/test/resources/logback-test.xml

```diff
@@ -13,17 +13,17 @@
         </encoder>
     </appender>
 
-    <logger name="org.constellation" level="info" />
+    <logger name="org.constellation" level="debug" />
```

Comment:
intentional changes? or do you forget about revert?

Reply:
Intentional, we've been talking about that to keep tests and cluster with debug enabled

Reply:
oki